packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: import_of_legacy_library_into_null_safe
+// ignore_for_file: import_of_legacy_library_into_null_safe, directives_ordering
```

Comment:
remove the ignore for directives_ordering, here and elsewhere. I just searched through this PR for "directives_ordering" to find all instances.